docs/internal_api.yml

```diff
@@ -512,6 +512,43 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+  /tenants/{tenant_id}/device/deployments/last:
```

Comment:
since we accept list of ids, shouldn't the path contain /devices/ instead of /device/ ?

Reply:
fixed.